esmvaltool/recipes/recipe_check_obs_FLUXCOM.yml

```diff
@@ -0,0 +1,28 @@
+# ESMValTool
```

Comment:
You should probably remove the whole recipe and add its parts to the [general](https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/recipes/examples/recipe_check_obs.yml) one.

Reply:
I don't have access to all the obs data sets in the general file on JASMIN I think. That's the reason I wrote one just for this data set.

Reply:
You can still use the general one for testing, simply commenting out the other datasets.